### DISCUSSION TITLE
How to delete sector task and retry do it

### DISCUSSION THREAD

**Discussion by zhangwangjin** [filecoin-project/lotus#7764] [Storage Provider] [answered]
Please provide all the information requested here to help us troubleshoot "commit failed" issues.
If the information requested is missing, we will probably have to just ask you to provide it anyway,
before we can help debug.

.sealing is stop.

A brief description of the problem you encountered while proving (sealing) a sector.

as lotus-worker node's disk is not enough, i delete /var/tmp/filecoin-proof-parameters and take other's matchine's  filecoin-proof-parameters to mount  /var/tmp/filecoin-proof-parameters, so the hard disk has been increased 100Gb, but sectors Compute ProofFailed,i wanna know how to delete this sectorts tasks and retry do it

Including what commands you ran, and a description of your setup, is very helpful.

**Sectors list**
![image](https://user-images.githubusercontent.com/9051261/87264676-2ed48c80-c4f3-11ea-9ea5-0ae9d0913c93.png)

The output of `./lotus-storage-miner sectors list`.

**Sectors status**
![image](https://user-images.githubusercontent.com/9051261/87264733-60e5ee80-c4f3-11ea-85fc-3559b57e7e28.png)

The output of `./lotus-storage-miner sectors status --log <sectorId>` for the failed sector(s).

{"level":"info","ts":"2020-07-13T10:27:31.009+0800","logger":"sectors","caller":"storage-fsm/states_sealing.go:253","msg":"scheduling seal proof computation..."}
{"level":"info","ts":"2020-07-13T10:27:31.009+0800","logger":"sectors","caller":"storage-fsm/states_sealing.go:255","msg":"KOMIT 7 0ef4d591dfb2f0124459b8cab3d16e85d3bdd63e2fbae212e5b393ba8b86c1f6(74258); 56838f8f2fb3113069dbb68f30577c2bd7ad5811ded7453900a3d8ab55d7fb5e(76199); [{34359738368 bafk4chzaa57f7xrvyufjga5fkae6gsmkj27n7444ik3rbnzq3dwhvr5puy7a}]; r:6261666b3465687a6175706d6f65783579706a736e7467783777637733623472786a6277776735636b363375366469626471623661726278766f356c71; d:6261666b3463687a6161353766377872767975666a676135666b61653667736d6b6a32376e37343434696b3372626e7a71336477687672357075793761"}
{"level":"info","ts":"2020-07-13T10:27:31.024+0800","logger":"sectors","caller":"storage-fsm/states_failed.go:19","msg":"ComputeProofFailed(7), waiting 59.975216111s before retrying"}
{"level":"warn","ts":"2020-07-13T10:27:31.298+0800","logger":"sectors","caller":"storage-fsm/fsm.go:364","msg":"sector 8 got error event sealing.SectorSealPreCommit1Failed: seal pre commit(1) failed: maybeSchedRequest didn't find any good workers"}
{"level":"info","ts":"2020-07-13T10:27:31.301+0800","logger":"sectors","caller":"storage-fsm/states_failed.go:19","msg":"SealPreCommit1Failed(8), waiting 59.698248424s before retrying"}
{"level":"info","ts":"2020-07-13T10:27:45.013+0800","logger":"sectors","caller":"storage-fsm/states_sealing.go:253","msg":"scheduling seal proof computation..."}
{"level":"info","ts":"2020-07-13T10:27:45.013+0800","logger":"sectors","caller":"storage-fsm/states_sealing.go:255","msg":"KOMIT 4 0faaeb2efe3fb15d4938557b7f1d213

Please go through the logs of your storage miner, and include screenshots of any error-like messages you find.

lotus-storage-miner version 0.4.1+git.ee7bdf38

The output of `./lotus --version`.

**Comment by hyunmoon** [comment]
I, too, don't know the correct way to handle it but I just remove related files and set Unsealed state on the sector.
I think `sectors remove` is meant to be used in this kind of situation but it has never worked for me so far.
`lotus-storage-miner update-state --really-do-it SECTOR_NUM Unsealed`

**Comment by Reiers** [comment, marked as answer]
The solution to this is
Run: `lotus-miner sealing jobs`
- Find the JOB ID
- Abort job with `lotus-miner sealing abort <ID>`
- `lotus-miner sectors remove --really-do-it <sectornumber>`
- if the sector have failed, but is active onChain then `lotus-miner sectors terminate --really-do-it <sectornumber>`

_**This issue is outdated, and solved in newer versions of Lotus.**_
